extensions/eclipse-che-theia-plugin-ext/package.json

```diff
@@ -20,7 +20,8 @@
     "@theia/mini-browser": "next",
     "@theia/plugin-ext": "next",
     "axios": "0.19.0",
-    "js-yaml": "3.13.1"
+    "js-yaml": "3.13.1",
+    "@eclipse-che/workspace-telemetry-client": "^0.0.1-1576775136"
```

Comment:
should it be `next` or `latest` there ?

Reply:
@davidfestal wdyt? I'm not very fluent with versioning best practice and we do publish neither `latest` nor `next` atm - https://www.npmjs.com/package/@eclipse-che/workspace-telemetry-client

Reply:
I assume we can use `latest`, and so manage it the same way as `@eclipse-che/workspace-client` is managed.

Reply:
only the `latest` tag is available according to what I see on the NPMJS versions page.

Reply:
bumped to the 'latest'